src/placeholder/placeholder.service.ts

```diff
@@ -30,7 +30,7 @@ export class PlaceholderService {
 		if (!this.viewContainerRef) {
 			console.error("No view container defined! Likely due to a missing `ibm-placeholder`");
 		}
-		return this.viewContainerRef.createComponent(componentFactory, 0, injector);
+		return this.viewContainerRef.createComponent(componentFactory, undefined, injector);
```

Comment:
I think we generally use `null` for skipping arguments ... unless `undefined` explicitly works best here?
```suggestion
		return this.viewContainerRef.createComponent(componentFactory, null, injector);
```

Reply:
oh sure @cal-smith, that makes sense to make it consistent with your practices